### PR TITLE
Terraform deployment edits

### PR DESCRIPTION
**Pull request by llatif**
Cloud SQL high traffic info commented out - further context needed

### CODE REVIEW

**Comment by crwilcox** [comment]
Please remove line 167 to around 182. That was some notation I had in while researching a reasonably small setup for this if folks wanted to scale down. None of it is really relevant to the deployment docs

File: docs/deploying.md [hunk 1]
```diff
@@ -69,119 +69,119 @@ Each service's `main` package is located in the `cmd` directory.
...
 
-1.  Create a GCP project.
-    [Instructions](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
-    Enable a billing account for this project, and remember its project ID (the
-    unique, unchangeable string that you will be asked for during creation).
-    
-    ```text
-    $ export PROJECT_ID="<value-from-above>"
-     ```
+   To install Terraform using Go, type:
```
Comment (sethvargo) [inline docs/deploying.md:80]: You should _not_ do this. Terraform has pre-built binaries. Don't instruct users to compile from source.

File: docs/deploying.md [hunk 2]
```diff
@@ -69,119 +69,119 @@ Each service's `main` package is located in the `cmd` directory.
...
+     -var repo_name=${YOUR_REPO_NAME}
+   ```
 
+Terraform will begin by creating the service accounts and enabling the services
+on Google Cloud that are required to run this server.
 
-30 gb which gives 500 concurrent connections
+#### High traffic example deployment
 
+**Important**: This is a production-ready, high traffic sample deployment.
```
Comment (sethvargo) [inline docs/deploying.md:152]: This is incorrect. The _default_ behavior is a a production-ready deployment. The example Terraform below is for a local development version. It's a smaller CloudSQL tier and database size.

File: docs/deploying.md [hunk 3]
```diff
@@ -69,119 +69,119 @@ Each service's `main` package is located in the `cmd` directory.
...
+     -var use_build_triggers=true \
+     -var repo_owner=${YOUR_REPO_OWNER} \
+     -var repo_name=${YOUR_REPO_NAME}
+   ```
 
+Terraform will begin by creating the service accounts and enabling the services
+on Google Cloud that are required to run this server.
 
-30 gb which gives 500 concurrent connections
+#### High traffic example deployment
```
Comment (crwilcox) [inline docs/deploying.md:150]:
```suggestion
#### Scaling down the size of the deployment
```

File: docs/deploying.md (hunk 2 above)
Comment (crwilcox) [inline docs/deploying.md:152]:
```suggestion
**Important**: The default configuration will create a production-ready, high traffic deployment.
```

Comment (llatif) [reply docs/deploying.md]: Thanks, edits applied.

File: docs/deploying.md (hunk 1 above)
Comment (llatif) [reply docs/deploying.md]: Removed as requested.

File: docs/deploying.md (hunk 2 above)
Comment (llatif) [reply docs/deploying.md]: Believe both of these edits have been incorporated in the latest branch commit. PTAL, cheers

File: docs/deploying.md [hunk 4]
```diff
@@ -69,119 +69,110 @@ Each service's `main` package is located in the `cmd` directory.
...
+     -var project=${PROJECT_ID} \
+     -var region="us-central-1" \
+     -var use_build_triggers=true \
+     -var repo_owner=${YOUR_REPO_OWNER} \
+     -var repo_name=${YOUR_REPO_NAME} \
+     -var cloudsql_tier="db-custom-1-3840" \
+     -var cloudsql_disk_size_gb="16"
+   ```
+
+1. Initialize or migrate the database.
```
Comment (crwilcox) [inline docs/deploying.md:161]: This is part of the broader steps. Is this getting grouped under the 'testing deployment'?

Comment (llatif) [reply docs/deploying.md:142]: Sorry misunderstood previous message, moved this into the main section, leaving the local dev and test section for only a scaled down deployment.

**Pull request merged**